rollup.config.js

```diff
@@ -7,9 +7,10 @@ import replace from 'rollup-plugin-replace';
 import packageJson from './package.json';
 
 const { name, version } = packageJson;
-const banner = `/* ${name} myron.liu version ${version} */`;
+const shortName = name.split('/').pop()
```

Comment:
`name` was eerst `"muse-ui"` maar is nu `"@jobport/muse-ui"`. Daarom hebben we nu dus dit stukje code nodig omdat we anders een extra subdirectory krijgen die `@jobport/` heet (zie regel 13, 45, etc.)